apps/teams-test-app/src/components/utils/ApiWithTextInput.tsx

```diff
@@ -55,7 +55,11 @@ export const ApiWithTextInput = <T extends unknown>(props: ApiWithTextInputProps
         }
       }
     } catch (err) {
-      setResult('Error: ' + err);
+      let error = `${err}`;
+      if (error === '[object Object]') {
+        error = JSON.stringify(err);
+      }
+      setResult('Error: ' + error);
```

Comment:
This code is duplicated in ApiWithoutInput.tsx. Can we make this a helper function that is called in both places?